_posts/2015-09-08-westerndevs-learn-about-docker-part-2.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "WesternDevs learn about Docker - Part 2"
+date: 2015-09-08 1:30:00
+author: dave_white
+originalurl: 
+comments: true
+---
+
+##### Dislaimer
+This blog post serves two purposes: Act as a historical record of a conversation with a bunch of interesting links in context, and to share a bit of an insider look at how conversations happen in the WesternDevs slack channels.
+
+### Introduction
+As Tom mentioned in his post about [Docker containers for Novices][1] which I'm considering Part 1 of this post, the WesternDevs had a conversation about how Docker containers work. A good portion of our group has a lot of experience in the Windows world and not a lot of *nix experience which is where containers seem to have been born from. 
+
+One of the things that I love about WesternDevs is the fairly voracious appetite that all of us have to understand the various technologies that we use and talk about while also ensuring that the whole group understands. Leave no man behind while we learn I guess you would say.  
+
+### The seed of the conversation
+"Can AD, DHCP, DNS run in a container?" seems like an innocuous question, but it was the question that started the conversation.
```

Comment:
Might be worth adding a link to where the question originated: http://codebetter.com/kylebaley/2015/09/01/windows-server-containers-are-coming-whether-you-like-it-or-not. Can't figure out how to link to a comment directly on there though.